code/modules/admin/verbs/debug.dm

```diff
@@ -313,58 +313,20 @@
 		return
 	do_dressing(H)
 
-/client/proc/do_dressing(var/mob/living/carbon/human/M = null)
+/proc/do_dressing(var/mob/living/carbon/human/M = null)
```

Comment:
Should probably type this as a proc of mob/living/carbon/human and use src, instead of as a global proc and passing in the mob?

Reply:
it's not a human proc because it works as a verb if no human is supplied:
`	if(!M || !istype(M))
		M = tgui_input_list(usr, "Select a mob you would like to dress.", "Set Human Outfit", GLOB.human_mob_list)
	if(!M)
		return
`